api/src/main/java/org/eclipse/microprofile/config/spi/Converter.java

```diff
@@ -24,7 +24,7 @@
  * <p>A Converter can specify a {@link javax.annotation.Priority}.
```

Comment:
I see your point, but my experiences with 250 are very mixed.

There are several conflicting dependencies with (parts of) its types out there, some are even in the JDK itself. So it's very prone to cause classpath conflicts, i.e. users will end up with the same types being pulled in multiple times from several dependencies. Which is esp. an issue when thinking about Java 9 / Jigsaw. Hence I'd try to avoid that dependency as much as possible.